source/extensions/tracers/zipkin/span_buffer.cc

```diff
@@ -137,7 +137,8 @@ JsonV2Serializer::toListOfSpans(const Span& zipkin_span, Util::Replacements& rep
       // us 1.58432429547687e+15. Instead we store it as the string of 1584324295476870 (when it is
       // serialized: "1584324295476870"), and replace it post MessageToJsonString serialization with
       // integer (1584324295476870 without `"`), see: JsonV2Serializer::serialize.
-      (*fields)[SPAN_TIMESTAMP] = Util::uint64Value(annotation.timestamp(), replacements);
+      (*fields)[SPAN_TIMESTAMP] =
+          Util::uint64Value(annotation.timestamp(), "timestamp", replacements);
```

Comment:
Can we have this "timestamp" as a constant?

Reply:
Yes. Done. Used `SPAN_TIMESTAMP` (and `SPAN_DURATION`).